.github/workflows/pages-preview.yml

```
@@ -0,0 +1,63 @@
+name: Deploy PR Preview
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)